tpm2/policy_v0.go

```diff
@@ -68,10 +69,184 @@ func computeV0PinNVIndexPostInitAuthPolicies(alg tpm2.HashAlgorithmId, updateKey
 	return out
 }
 
+type policyOrDataNode_v0 struct {
```

Comment:
it feels like this is v0 but shared forward so maybe it belongs at the top of the file after imports instead of having first a v0 specific bit